src/open_deep_research/graph.py

```
@@ -78,10 +82,33 @@ async def generate_report_plan(state: ReportState, config: RunnableConfig):
+                model_provider="openai",
+                model_kwargs={"openai_api_key": openai_api_key}
+            )
+        elif ':' in writer_model_name:
+            writer_provider, model_name = writer_model_name.split(':', 1)
+            writer_model = init_chat_model(model=model_name, model_provider=writer_provider, model_kwargs=writer_model_kwargs)
+        else:
+            writer_provider = get_config_value(configurable.writer_provider)
+            writer_model = init_chat_model(model=writer_model_name, model_provider=writer_provider, model_kwargs=writer_model_kwargs)
+    except Exception as e:
+        # Fallback to Groq if there's an error with OpenAI
+        print(f"Error initializing OpenAI model: {e}. Falling back to Groq.")
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)